test/cql-pytest/cassandra_tests/validation/operations/alter_test.py

```diff
@@ -206,8 +206,8 @@ def testCreateAlterKeyspaces(cql, test_keyspace, this_dc):
 # Reproduces CASSANDRA-12681 and Scylla #10036
 def testAlterKeyspaceWithNoOptionThrowsConfigurationException(cql, test_keyspace, this_dc):
     # Create keyspaces
-    with create_keyspace(cql, "replication={ 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 3 }") as abc:
-        with create_keyspace(cql, "replication={ 'class' : 'NetworkTopologyStrategy', 'replication_factor' : 3 }") as xyz:
+    with create_keyspace(cql, "replication={ 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 3 }", force_no_tablets=True) as abc:
+        with create_keyspace(cql, "replication={ 'class' : 'NetworkTopologyStrategy', 'replication_factor' : 3 }", force_no_tablets=True) as xyz:
```

Comment:
In my PR, I made this test fails_with_tablets - because it demonstrates a real difference between what Cassandra allows, and Scylla used to allow and no longer does.
But I won't insist. I also wasn't sure what exactly to do about this test.

Reply:
Yes, and any test which works only with tablets disabled, has an expiry date on it, as we want to retire vnodes at one point in the future. So any such workarounds, are temporary by nature.